automation/release/send-zip-to-s3.sh

```diff
@@ -62,42 +62,81 @@ cp ${API_POM} ${TEMPDIR}/newrelic-api/${VERSION}/newrelic-api-${VERSION}.pom
 
 EXITSTATUS=0
 
+# when running on GHA, we cannot specify the profile, like we do on our managed machines.
+# Auth is done thru the aws-actions/configure-aws-credentials action.
+if [ -z ${RUNNING_ON_GHA} ];
+then
+  echo "Running locally"
+  PROFILE="--profile ${S3_PROFILE}"
+  OUTPUT=/dev/stdout
+else
+  echo "Running on Gha"
+  OUTPUT=$GITHUB_STEP_SUMMARY
+fi
+
+# Sync agent's $TEMPDIR to s3
```

Comment:
the aws commands below are essentially the same, but with the ${PROFILE} envvar, so no profile is set when running on GHA.